content/workshops/rhel_8/exercise1.0.adoc

```diff
@@ -0,0 +1,236 @@
+workshops: rhel_8
+workshop_weight: 10
+layout: lab
+---
+
+:domain_name: redhatgov.io
+:icons: font
+:imagesdir: /workshops/rhel_8/images
+
+
+= Exercise 1.0 - Welcome to RHEL!
+
+
+== Exercise Description
+
+Welcom to RHEL!
+
+Red Hat Enterprise Linux 8 is the intelligent enterprise Linux, providing a secure and consistent foundation across hybrid cloud deployments and the tools enterprises need to deliver services and workloads faster with less effort.  RHEL 8 enables delivery of any application, on any footprint, at any time.
+
+With predictable release cadence and 10-year enterprise support, RHEL 8 is the safe and stable choice for your enterprise.
+
+RHEL 8 also automates regulatory compliance and security configuration remediation ​across your system and within containers​​ with OpenSCAP, Red Hat’s National Institute of Standards and Technology (NIST)-certified scanner.  RHEL 8's security portfolio also includes a new, system-wide encryption policy and the latest nftables/firewalld firewall.
+
+To get started, we will take a tour of the web console. The web console (called `cockpit`) will be your primary interactive interface to RHEL. The console has a number of functions that allow you to manage most aspects of the installation.
+
+=== Section 1: Overview of the web console
+
+Here is a quick overview of all the available modules:
+
+[.float-group]
+--
+image::cockpit_topics.png[caption="Figure 1: ", title="Web console functions", width=450, float=left]
+The various functional sections can be seen in `Figure 1`, and are: +
+{empty} +
+*System* +
+Contains an overview of the system, including metadata and health graphs
+{empty} +
+*Logs* +
+Displays current `journald` logs from all services
+{empty} +
+*Storage* +
+Displays storage activity graphs, and logs. Allows for the management of NFS mounts, RAID devices, volume groups, VDO (deduplication) devices, and iSCSI targets
+{empty} +
+*Networking* +
+Displays network activity, logs, and allows firewall and network maintenance
+{empty} +
+*Podman Containers* +
+Allows for the management of container images and running containers
+{empty} +
+*Virtual Machines* +
+Virtual machines can be created, deleted and controlled
+{empty} +
+*Accounts* +
+Allows for the creation, deletion and configuration of user accounts and ssh keys
+{empty} +
+*Services* +
+Displays all system targets, services, sockets, timers and paths, as managed by `systemd`, the root process
+{empty} +
+*Session Recording* +
+Manages recorded sessions, and allows for the configuration of session recorder defaults
+{empty} +
+*Applications* +
+Allows for the installation and removal of web console modules
+{empty} +
+*Diagnostic Reports* +
+Creates diagnostic reports used by Red Hat support, also known as SOSreports
+{empty} +
+*Kernel Dump* +
+Manages kernel debugging dumps, created when the system crashes
+{empty} +
+*SELinux* +
+Allows for the setting of SELinux to permissive mode, and shows recent SELinux alerts (`AVCs`)
```

Comment:
Line 74: Enables SELinux to be set to Passive mode and displays recent SELinux alerts (AVCs)